### ISSUE TITLE
multiple sensors identification issue

### ISSUE THREAD

**Issue by dorinvlas** [mishop/sensors#2]
Hi Misho, 
I found out that your go app works perfectly as sensors integrated in telegram, its just that identification does not work.

When one or more sensors from a set of lets say 3 is disconnected and then reconneced after a few seconds, the naming of the sensors will change and the reporting via telegraf will be wrong.

I think some sort of unique naming with bind to unique 1wire address is needed.

Thanks for all your efforts.
Regards, 
DV

**Comment by mishop** [comment]
Yes you are right. I thought about solving this problem before. Unfortunately lightning killed my raspberry pi, so I can't test. I'll try to think of something.

**Comment by DarthBlair** [comment]
Hi, is there any progress on that topic?

Could a solution be to simply sort the list from:
```go
data, err := ioutil.ReadFile("/sys/bus/w1/devices/w1_bus_master1/w1_master_slaves")

sensors := strings.Split(string(data), "\n")
```
